docs/crs.md

```diff
@@ -0,0 +1,54 @@
+# Coordinate Reference System (CRS)
+
+When running a populator implementation, the STAC populator assumes that the metadata from the files will specify
+which CRS is used to represent the geospatial coordinates in the data.
+
+For example, The `THREDDSPopulator` parses metadata from NCML metadata and looks for values named
+`geospatial_bounds_crs` (and optionally `geospatial_bounds_crs_vertical`) to determine the CRS.
+
+If these values are not specified then the CRS is unknown and STAC populator will not know how to convert the
+geospatial coordinates to a CRS that STAC accepts. If nothing is specified, then STAC populator will assume
+that the CRS is EPSG:4326 (for 2D coordinates) or EPSG:4979 (for 3D coordinates).
```

Comment:
Provide the links for quick reference (eg: https://epsg.io/4326)

Consider also adding the `WGS84` reference as alias in case someone is not familiar with its equivalency with the EPSG code.